Sources/SkeletonView.swift

```diff
@@ -94,6 +94,7 @@ public extension UIView {
 
 extension UIView {
     @objc func skeletonLayoutSubviews() {
+        guard Thread.isMainThread else { return }
```

Comment:
I think this guard should be after the `skeletonLayoutSubviews` method. Here, the original `layoutSubviews` method will not be called.

Reply:
The stacktrace shows us that the issue came from the skeletonLayoutSubviews. We should "kill" it as soon as possible, so in the first call, what do you think ?
In any case, this should not be called on another thread but the main one.